src/renderer/families/cosmos/Delegation/index.js

```diff
@@ -0,0 +1,228 @@
+// @flow
+import React, { useCallback } from "react";
+import invariant from "invariant";
+import { useDispatch } from "react-redux";
+import { Trans } from "react-i18next";
+import styled from "styled-components";
+import type { Account } from "@ledgerhq/live-common/lib/types";
+import { getAccountUnit } from "@ledgerhq/live-common/lib/account";
+import { formatCurrencyUnit } from "@ledgerhq/live-common/lib/currencies";
+import { useCosmosPreloadData } from "@ledgerhq/live-common/lib/families/cosmos/react";
+
+import type {
+  CosmosDelegation,
+  CosmosValidatorItem,
+  CosmosDelegationStatus,
+} from "@ledgerhq/live-common/lib/families/cosmos/types";
+import type { BigNumber } from "bignumber.js";
+
+import { urls } from "~/config/urls";
+import { openURL } from "~/renderer/linking";
+import { openModal } from "~/renderer/actions/modals";
+import Text from "~/renderer/components/Text";
+import Button from "~/renderer/components/Button";
+import Box, { Card } from "~/renderer/components/Box";
+import LinkWithExternalIcon from "~/renderer/components/LinkWithExternalIcon";
+import IconChartLine from "~/renderer/icons/ChartLine";
+import Header from "./Header";
+import Row from "./Row";
+
+import ToolTip from "~/renderer/components/Tooltip";
+import ClaimRewards from "~/renderer/icons/ClaimReward";
+import { useDiscreetMode } from "~/renderer/components/Discreet";
+
+/** @TODO move this in common */
+const formatDelegations = (
+  delegations: CosmosDelegation[],
+  validators: CosmosValidatorItem[],
+): {
+  validator: ?CosmosValidatorItem,
+  address: string,
+  pendingRewards: BigNumber,
+  amount: BigNumber,
+  status: CosmosDelegationStatus,
+}[] => {
+  return delegations.map((d, i, arr) => ({
+    validator: validators.find(v => v.validatorAddress === d.validatorAddress),
+    address: d.validatorAddress,
+    amount: d.amount,
+    pendingRewards: d.pendingRewards,
+    status: d.status,
+  }));
+};
+
+type Props = {
+  account: Account,
+};
+
+const Wrapper = styled(Box).attrs(() => ({
+  p: 3,
+  mt: 24,
+  mb: 6,
+}))`
+  border: 1px dashed ${p => p.theme.colors.palette.text.shade20};
+  border-radius: 4px;
+  justify-content: space-between;
+  align-items: center;
+`;
+
+const Delegation = ({ account }: Props) => {
+  const dispatch = useDispatch();
+
+  const unit = getAccountUnit(account);
+
+  const { validators } = useCosmosPreloadData();
+
+  const { cosmosResources } = account;
+  invariant(cosmosResources, "cosmos account expected");
+  const { delegations, pendingRewardsBalance: _pendingRewardsBalance } = cosmosResources;
+
+  const discreet = useDiscreetMode();
+
+  const pendingRewardsBalance = formatCurrencyUnit(unit, _pendingRewardsBalance, {
```

Comment:
same feedback. 
```
  const { delegations, pendingRewardsBalance: _pendingRewardsBalance } = cosmosResources;
```
is boilerplate we could avoid by just directlying inlining the stuff.